runtime/common/src/crowdloan.rs

```diff
@@ -322,7 +322,8 @@ decl_module! {
 			origin,
 			#[compact] index: FundIndex,
 			#[compact] value: BalanceOf<T>,
-			signature: Option<MultiSignature>
+			signature: Option<MultiSignature>,
+			_memo: [u8; 32],
```

Comment:
why fixed size memo? it will be a wasted space for people not using it

Reply:
Good point. Maybe it should be `Vec<u8>` and we can enforce a maximum size.

Reply:
Took care of that. Thanks